members/README.md

```diff
@@ -91,3 +91,67 @@ Returns all inactive/idle members in the system.
 - **Error Response:**
   - **Code:** 500
     - **Content:** `{ 'statusCode': 500, 'error': 'Internal Server Error', 'message': 'Something went wrong. Please contact admin' }`
+
+
+
+------
+# Getting Started with Recruiter Introductory Forms
+
+## **Recruiter Object**
+
+```
+{
+    'company': string,
+    'fname': string,
+    'lname': string,
+    'designation': string,
+    'reason': string,
+    'email': string,
+    'currency': string
+    'package': number
+}
+```
+## **Requests**
+
+|Route|Description|
+|:---:|:---------:|
+|POST /members/intro|Post request for members profile introduction|
```

Comment:
There's no need to create this table again, we already have this at line no 25. Why not just add an extra row for this request and link it.